jussi/serve.py

```diff
@@ -2,6 +2,7 @@
 import argparse
 import os
 
+import asyncio
```

Comment:
There must be a big performance drop by not using uvloop and worker processes. I'll create an issue after merging this PR as a reminder to start using them again.